test/Traveler-test.js

```diff
@@ -0,0 +1,32 @@
+import { expect } from 'chai';
+
+import Traveler from '../src/Traveler';
+import travelerData from '../src/test-data/Traveler-test-data';
+import tripsData from '../src/test-data/Trip-test-data';
+
+describe('Traveler', () => {
+  let traveler;
+
+  beforeEach(() => {
+    traveler = new Traveler(travelerData[0], tripsData);
+  });
+
+  it('should be a function', () => {
+    console.log(traveler)
+  })
+
+  // it('is a function', () =>{
+  //   expect(Traveler).to.be.a('function');
+  // });
+  //
+  // it('should be an instance of Traveler', () => {
+  //   expect(traveler).to.be.an.instanceof(Traveler);
+  // });
```

Comment:
?? Did you want to comment these back in for testing porpoises?
![porpoise](https://media.giphy.com/media/l397265bGSRxERETm/giphy.gif)

Reply:
Yep! I did after I initially pushed this. haahah Thank you